table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GluePartition.java

```diff
@@ -45,4 +45,30 @@ public String getSpec() {
   public Layout getLayout() {
     return mLayout;
   }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
```

Comment:
Is there a reason we can't use `MoreObjects.toStringHelper()` for toString? Also, I don't think we have to check for null before adding fields, since it will just print null in the output, right?

Reply:
I see, I just remove the check for null and changed to `MoreObjects.toStringHelper()`.